### ISSUE TITLE
Accessing inventory remotely.

### ISSUE THREAD

**Issue by ndftx** [thexaero/open-parties-and-claims#498]
Using Simple Network Storage, when I claim my own chunks, it works fine wherever in the server I am, When my friend claims the chunks it only works while he's in the server and when he leaves, then his forceloaded chunks go unloaded. Even turning on offline forceload, this does not fix the issue. How can his forceloads continue to live on while he is offline?

**Comment by thexaero** [comment]
What config did you enable offline forceloads in? Also, please try with the whole network being forceloaded. I haven't used Simple Network Storage specifically but a similar mod had an issue like this caused by not everything necessary being forceloaded.

**Comment by thexaero** [comment]
Does it work now?

**Comment by ndftx** [comment]
> Does it work now?

Yes it does, I had thought I posted a reply when I fixed my issue and corrected to completed. My fault

For the mod pack BetterMC, it doesn't let you change players configs by default, so once I turned on force loading in the server.toml I had to make sure I could enable force loading for each player that needed it. Once I set my friend's config, it updated all force load tickets and ended up working. I can go into detail for future people that have a similar issue if you'd like
